src/core/libraries/VestingMath.sol

```diff
@@ -99,19 +99,21 @@ library VestingMath {
     /// @dev Lockup linear model uses the following distribution function:
     ///
     /// $$
-    /// f(x) = x * d + c
+    /// f(x) = x * sa + s + c
```

Comment:
I'd prefer `s + c + x * sa`, the order in which they are unlocked.

Reply:
since it is a mathematical notation (function of time - x), i believe keeping like this  is the correct choice

in mathematics, expressions are typically written with variable-dependent and higher-degree terms first, followed by constants, to highlight the function’s main behavior and make its structure immediately clear

Reply:
Oh you are right. How could I forget: `y = mx + c`.